py-polars/tests/unit/operations/map/test_inefficient_map_warning.py

```diff
@@ -62,18 +62,37 @@
         'pl.col("a").is_in((1, 2, 3, 4, 3)) & ((pl.col("a") % 2) == 0) & (pl.col("a") > 0)',
     ),
     ("a", "lambda x: MY_CONSTANT + x", 'MY_CONSTANT + pl.col("a")'),
+    (
+        "a",
+        "lambda x: (float(x) * int(x)) // 2",
+        '(pl.col("a").cast(pl.Float64) * pl.col("a").cast(pl.Int64)) // 2',
+    ),
+    # ---------------------------------------------
+    # numpy
+    # ---------------------------------------------
```

Comment:
I've just reordered the tests a bit here, to make a new "numpy" section - that involved moving this test (`float(x) * int(x)`) up a bit